packages/web/src/pages/UserInterface/index.tsx

```diff
@@ -29,8 +30,20 @@ type UserInterface = {
   };
 };
 
+const defaultPrimaryMainColor = '#0059F7';
+const defaultPrimaryDarkColor = '#001F52';
+const defaultPrimaryLightColor = '#4286FF';
```

Comment:
These values are duplicated as they reside in the default theme. So, you should utilize the valuers from `packages/web/src/styles/theme.ts`.